src/Common/ExecCommand.php

```diff
@@ -220,28 +124,29 @@ protected function useCallOnWindows($cmd)
         return $cmd;
     }
 
+    protected function getCommandDescription()
+    {
+        return $this->getCommand();
+    }
+
+    public function getCommand()
```

Comment:
Can `getCommand()` simply be removed from `ExecTrait` now?  Make `getCommandDescription()` call `$this->process->getCommandLine()`, but first test to see if `$this->process` has been initialized yet, and return some default "uninitialized" string if not. (Guard against incorrect usage.)